netsim/modules/evpn.py

```diff
@@ -127,6 +181,9 @@ def vrf_transit_vni(topology: Box) -> None:
       continue  
     vni_list.append( vni )                                      # Insert it to detect duplicates elsewhere
 
+  if vni_error:                                                 # Found serious errors, makes no sense to continue
```

Comment:
if evpn_transport != "vxlan" also return, no vnis to assign

Reply:
While not exactly mandatory, added code to count VRFs with evpn.transit_vni and exit early if there are none -- let's reduce the carbon footprint a bit ;)

Thank you!